core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java

```diff
@@ -105,11 +108,13 @@ public void restorePositiveItem(String itemReference) {
       Node node = session.getNodeByIdentifier(itemReference);
       String fileName = node.getName();
       restoreFromQuarantine("/" + DLP_SECURITY_FOLDER+ "/" + fileName, WCMCoreUtils.getUserSessionProvider());
+      indexingService.reindex(TYPE, itemReference);
```

Comment:
I'm not sure we have to explicitly reindex it here. 
The reindexation should rerun automatically when the item is moved to original folder. My question is : we should ensure that it is really the case.